### ISSUE TITLE
Playing stops after each song from a playlist

### ISSUE THREAD

**Issue by duhanebel** [volumio/volumio-plugins#164]
When listening to a "daily mix playlist" the playing stops at the beginning of each new song.
From the logs I can't see what's wrong

`2018-03-10T09:27:15.296Z - info: [SpConDebug] SpotConn is playing
2018-03-10T09:27:15.311Z - info: CoreStateMachine::setConsumeUpdateService undefined
2018-03-10T09:27:15.316Z - info: CoreCommandRouter::servicePushState
2018-03-10T09:27:15.319Z - info: CoreStateMachine::syncState
2018-03-10T09:27:15.322Z - info: CoreStateMachine::pushState
2018-03-10T09:27:15.325Z - info: CoreStateMachine::getState
2018-03-10T09:27:15.351Z - info: CoreCommandRouter::volumioPushState
2018-03-10T09:27:15.354Z - info: CoreCommandRouter::executeOnPlugin: volumiodiscovery , saveDeviceInfo
2018-03-10T09:27:15.383Z - info: interfaceApi::pushState
2018-03-10T09:27:15.538Z - info: Pushing Favourites {"service":"volspotconnect2","uri":"spotify:track:2f13c461a1bf4877beb64899fe642e98","favourite":false}
===============The new song is queued and paused =======================
========== Nothing else happens but if I press PLAY again ====================
2018-03-10T09:31:39.644Z - info: [SpConDebug] Killing Volumio state, Spotify session: true
2018-03-10T09:31:39.659Z - info: [SpConDebug] SpotConn is done!
2018-03-10T09:31:39.662Z - info: [SpConDebug] Received stop
2018-03-10T09:31:39.733Z - info: [SpConDebug] Killing Volumio state, Spotify session: true
2018-03-10T09:31:39.746Z - info: CoreStateMachine::resetVolumioState
2018-03-10T09:31:39.756Z - info: CoreStateMachine::getcurrentVolume
2018-03-10T09:31:39.766Z - info: CoreCommandRouter::volumioRetrievevolume
2018-03-10T09:31:39.882Z - info: CoreStateMachine::updateTrackBlock
2018-03-10T09:31:39.889Z - info: CorePlayQueue::getTrackBlock
2018-03-10T09:31:39.905Z - info: CoreCommandRouter::volumioStop
2018-03-10T09:31:39.914Z - info: CoreStateMachine::stop
2018-03-10T09:31:39.936Z - info: CoreStateMachine::setConsumeUpdateService undefined
2018-03-10T09:31:39.953Z - info: [SpConDebug] Token: [...]
2018-03-10T09:31:39.982Z - info: [SpConDebug] Session is active!
2018-03-10T09:31:39.988Z - info: CoreCommandRouter::volumioGetState
2018-03-10T09:31:39.995Z - info: CoreStateMachine::getState
2018-03-10T09:31:40.001Z - info: CorePlayQueue::getTrack 0
2018-03-10T09:31:40.008Z - info: CoreCommandRouter::volumioStop
2018-03-10T09:31:40.011Z - info: CoreStateMachine::stop
2018-03-10T09:31:40.018Z - info: CoreStateMachine::setConsumeUpdateService undefined
2018-03-10T09:31:40.253Z - info: VolumeController:: Volume=83 Mute =false
2018-03-10T09:31:40.265Z - info: CoreStateMachine::pushState
2018-03-10T09:31:40.272Z - info: CoreStateMachine::getState
2018-03-10T09:31:40.275Z - info: CorePlayQueue::getTrack 0
2018-03-10T09:31:40.285Z - info: CoreCommandRouter::volumioPushState
2018-03-10T09:31:40.292Z - info: CoreCommandRouter::executeOnPlugin: volumiodiscovery , saveDeviceInfo
2018-03-10T09:31:40.302Z - info: interfaceApi::pushState
2018-03-10T09:31:40.528Z - info: Pushing Favourites {"service":"spop","uri":"spotify:track:55z3vUzYpOnItyIzOYY8y8","favourite":false}`
2018-03-10T09:31:42.212Z - info:
---------------------------- Client requests Volumio stop
2018-03-10T09:31:42.237Z - info: CoreCommandRouter::volumioStop
2018-03-10T09:31:42.260Z - info: CoreStateMachine::stop
2018-03-10T09:31:42.276Z - info: CoreStateMachine::setConsumeUpdateService undefined
2018-03-10T09:31:42.471Z - info: [SpConDebug] SpotConn is playing
2018-03-10T09:31:42.484Z - info: CoreStateMachine::setConsumeUpdateService undefined
2018-03-10T09:31:42.487Z - info: CoreCommandRouter::servicePushState
2018-03-10T09:31:42.490Z - info: CoreStateMachine::syncState
2018-03-10T09:31:42.493Z - info: CoreStateMachine::pushState
2018-03-10T09:31:42.534Z - info: CoreStateMachine::getState
2018-03-10T09:31:42.550Z - info: CoreCommandRouter::volumioPushState
2018-03-10T09:31:42.554Z - info: CoreCommandRouter::executeOnPlugin: volumiodiscovery , saveDeviceInfo
2018-03-10T09:31:42.576Z - info: interfaceApi::pushState
2018-03-10T09:31:42.776Z - info: Pushing Favourites {"service":"volspotconnect2","uri":"spotify:track:2f13c461a1bf4877beb64899fe642e98","favourite":false}
2018-03-10T09:31:42.953Z - info: CoreCommandRouter::servicePushState
2018-03-10T09:31:42.960Z - info: CoreStateMachine::syncState
2018-03-10T09:31:42.967Z - info: CoreStateMachine::pushState
2018-03-10T09:31:42.970Z - info: CoreStateMachine::getState
2018-03-10T09:31:42.973Z - info: CoreCommandRouter::volumioPushState
2018-03-10T09:31:42.976Z - info: CoreCommandRouter::executeOnPlugin: volumiodiscovery , saveDeviceInfo
2018-03-10T09:31:42.993Z - info: interfaceApi::pushState
2018-03-10T09:31:43.264Z - info: [SpConDebug] Token: [...]
2018-03-10T09:31:43.369Z - info: [SpConDebug] Killing Volumio state, Spotify session: true
2018-03-10T09:31:43.381Z - info: [SpConDebug] SpotConn is done!
2018-03-10T09:31:43.386Z - info: [SpConDebug] Received stop
2018-03-10T09:31:43.416Z - info: [SpConDebug] Killing Volumio state, Spotify session: true
2018-03-10T09:31:43.425Z - info: CoreStateMachine::resetVolumioState
2018-03-10T09:31:43.432Z - info: CoreStateMachine::getcurrentVolume
2018-03-10T09:31:43.438Z - info: CoreCommandRouter::volumioRetrievevolume
2018-03-10T09:31:43.540Z - info: CoreStateMachine::updateTrackBlock
===============The next song plays =======================

**Comment by balbuze** [comment]
yes, the problem occur with the last update.. We are working to solve this asap.
Waiting the fix, you can use the dev version, installed via ssh : https://github.com/balbuze/volumio-plugins/tree/master/plugins/music_service/volspotconnect2
